library/spiders/books.py

```diff
@@ -0,0 +1,82 @@
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.devtools.v85.browser import close
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self):
+        super().__init__()
+        self._driver = webdriver.Chrome()
```

Comment:
The use of `webdriver.Chrome()` directly in the constructor can lead to performance issues, as each instance of the spider will create a new browser instance. Consider using a headless browser or managing the WebDriver instance more efficiently.